deploy/gk-deploy

```diff
@@ -239,7 +239,7 @@ while [[ $# -ge 1 ]]; do
   shift
 done
 
-if [[ ! -f ${TOPOLOGY} ]]; then
+if [[ ${ABORT} -ne 1 ]] && [[ ! -f ${TOPOLOGY} ]]; then
```

Comment:
Will it be useful. If I move to down, We will be asking for Namespace, cli and add template and then we will be quiting for topology file not found. I don't see a point why go through all these and then quit for topology file.